openid-sharedsignals-framework-1_0.md

```diff
@@ -1229,6 +1229,12 @@ Errors are signaled with HTTP status codes as follows:
 {: title="Create Stream Errors" #tablecreatestream}
 
 
+##### Validating a Stream Creation Response
+
+* `aud`: the Receiver SHOULD validate the `aud` in the Create Stream Response.
+A Transmitter and Receiver MAY agree upon the audience value out of band.
```

Comment:
The formatting doesn't look right. Have you tried making it locally and checking the output?

Reply:
Looks ok? 
<img width="778" alt="Screenshot 2025-04-24 at 10 09 35 AM" src="https://github.com/user-attachments/assets/a2b62663-d3f2-4ab2-b244-76a739704537" />